package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "generate": "lerna run generate",
     "postgenerate": "yarn lint:write",
+    "preinstall": "yarn config set ignore-engines true",
```

Comment:
prool requires node >= 22, but vercel only goes as high as 20 and the ui-demo app doesn't deploy with node engine 20... so we just hope prool runs on 18 🤷